### DISCUSSION TITLE
the moon's rising and setting times and Illustrations for it

### DISCUSSION THREAD

**Discussion by abusabda** [Stellarium/stellarium#1784] [Q&A]
I think, in Stellarium, the moon's rising and setting times should be corrected immediately. as well as illustrations for it. you can refer to Jean Meuss's book "Astronomical Alghorithm" for this. Thank you.

**Comment by gzotti** [comment]
What do you refer to? Which illustrations should be corrected?

**Reply by abusabda** [reply]
I think this issue has been discussed here

https://github.com/Stellarium/stelarium/issues/948

But until now there has been no correction for this problem.

**Reply by gzotti** [reply]
This issue has been reported, and I suggested Meeus elsewhere. It is in the "open" state, and we know about it. When we find some spare time, we will hopefully be able to solve it. No need to further discuss or even demand a fix, unless YOU provide it. And I still don't know which illustrations you refer to.

**Reply by abusabda** [reply]
The illustration I mean is when the moon rises and sets.

When the moon rises and sets according to Meuus, it is when the upper limb of the moon touches the horizon.
